src/app/components/address-input/address-input.component.spec.ts

```diff
@@ -314,4 +318,15 @@ describe('AddressInputComponent', () => {
             expect(component.value).toBe(firstAddress.address);
         });
     });
+
+    describe('as a programmatic input', () => {
```

Comment:
Can you also add a case where you add a random string programmatically and check if you get a proper validation error?

Reply:
Yes, that is helpful. Thanks!